app/src/main/res/values/strings.xml

```diff
@@ -1229,4 +1229,10 @@ Alternatively, you can leave a note (with a photo)."</string>
     <string name="quest_road_width_title">"What’s the width of this road here?"</string>
     <string name="quest_road_width_explanation">"The roadway width from curb to curb: This includes anything on the street surface like on-street parking or cycle lanes but excludes anything beyond it like cycle paths or parking beside the roadway."</string>
 
+    <string name="quest_seating_name_title">Does this place have seating?</string>
+    <string name="quest_seating_takeaway">Takeaway only</string>
+    <string name="quest_seating_indoor_only">Indoor seating only</string>
+    <string name="quest_seating_outdoor_only">Outdoor seating only</string>
+    <string name="quest_seating_indoor_and_outdoor">Both indoor and outdoor seating</string>
+    <string name="default_disabled_msg_summer_outdoor_seating">"This quest type is disabled by default because it can only be solved in summer season in order to check outdoor seating."</string>
```

Comment:
```suggestion
    <string name="default_disabled_msg_summer_outdoor_seating">"This quest type is disabled by default because it may require checking during specific part of the year in order to check outdoor seating."</string>
```

Not entirely sure how to phrase it better to be clear also in places where outdoor seating is present during other part of the year or all the time.